src/modules/components/vote/voteDataListItem/voteDataListItemStructure/voteDataListItemStructure.test.tsx

```diff
@@ -0,0 +1,96 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import { getAddress, isAddress } from 'viem';
+import { useAccount } from 'wagmi';
+import { DataList, NumberFormat, formatterUtils } from '../../../../../core';
+import { addressUtils } from '../../../../utils';
+import { VoteDataListItemStructure, type IVoteDataListItemStructureProps } from '../../voteDataListItem';
+
+jest.mock('wagmi', () => ({
+    useAccount: jest.fn(),
+}));
+
+jest.mock('viem', () => ({
+    isAddress: jest.fn(),
+    getAddress: jest.fn(),
+}));
+
+jest.mock('../../../../../core/components/tag', () => ({
+    Tag: ({ label }: { label: string }) => <div data-testid="tag">{label}</div>,
+}));
+
+jest.mock('../../../member', () => ({
+    MemberAvatar: () => <div data-testid="member-avatar" />,
+}));
+
+describe('<VoteDataListItemStructure /> component', () => {
+    const createTestComponent = (props?: Partial<IVoteDataListItemStructureProps>) => {
+        const completeProps: IVoteDataListItemStructureProps = {
+            voter: { address: '0x1D03D98c0aac1f83860cec5156116FE68725642E' },
+            voteIndicator: 'yes',
+            ...props,
+        };
+
+        return (
+            <DataList.Root entityLabel="Votes">
+                <DataList.Container>
+                    <VoteDataListItemStructure {...completeProps} />
+                </DataList.Container>
+            </DataList.Root>
+        );
+    };
+
+    beforeEach(() => {
+        (isAddress as unknown as jest.Mock).mockImplementation(() => true);
+        (getAddress as jest.Mock).mockImplementation((address: string) => address);
```

Comment:
Using the `jest.spyOn` functionality allows us to remove some casts, something like:
```
import * as viem from 'viem';

const isAddressSpy = jest.spyOn(viem, 'isAddress');

isAddressSpy.mockReturnValue(true);
```